### PR TITLE
Added new functions

### PR DESCRIPTION
**Pull request by dmytro-lt**
https://app.asana.com/0/1157793729216339/1201246276117419

### CODE REVIEW

File: src/twig.functions.js [hunk 1]
```diff
@@ -312,7 +313,23 @@ export default function (Twig) {
...
+
+            let utc = a.getTime() + (a.getTimezoneOffset() * 60000)
+            let b = new Date(utc + (36000 * offset))
+
+            return "The local time " + city + " is" + b.toLocaleDateString()
+        },
+        
+        include(Twig) {
+            
+            extends по всьому проекту в одному місці
```
Comment (andriy-kozlytski) [inline src/twig.functions.js]: What is this?

File: src/twig.functions.js [hunk 2]
```diff
@@ -312,7 +313,23 @@ export default function (Twig) {
...
+        },
+
+        country_timezones(city, offset) {
+
+            let a = new Date()
+
+            let utc = a.getTime() + (a.getTimezoneOffset() * 60000)
+            let b = new Date(utc + (36000 * offset))
+
+            return "The local time " + city + " is" + b.toLocaleDateString()
```
Comment (andriy-kozlytski) [inline src/twig.functions.js]: Must return international name of time zone by country

File: src/twig.functions.js [hunk 3]
```diff
@@ -312,7 +313,32 @@ export default function (Twig) {
...
+
+            var reader = new FileReader()
+
+            if(filetype != "twig") {
+                return false
+            } else {
+                return reader.onload = () => reader.result
+            }
+
+            return filetype
```
Comment (andriy-kozlytski) [inline src/twig.functions.js:339]: This code never execute

File: src/twig.functions.js [hunk 4]
```diff
@@ -312,7 +313,32 @@ export default function (Twig) {
...
+        
+        include(file) {
+            let filetype = file.split('.').pop()
+
+            var reader = new FileReader()
+
+            if(filetype != "twig") {
+                return false
+            } else {
+                return reader.onload = () => reader.result
```
Comment (andriy-kozlytski) [inline src/twig.functions.js]: It is async operation.
Where is "readAs" ?
reader.onload  - is  listener. This  callback must have  params with result

**Review by andriy-kozlytski — CHANGES_REQUESTED** [review]

File: src/twig.functions.js (hunk 1 above)
Comment (dmytro-lt) [reply src/twig.functions.js]: > What is this?

It was without saving. The last commit is actual

**Review by lodry — CHANGES_REQUESTED** [review]
ВИДАЛИ ЗМІНИ З ІНШИХ ГІЛОК 
 MT проконтролюй

**Pull request closed without merging**